package.yaml

```diff
@@ -94,6 +94,7 @@ dependencies:
   - binary
   - bcrypt
   - bytestring
+  - containers
```

Comment:
added for `Data.Sequence` for web request Fixtures

Reply:
See note on that file